modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+|===
+
+[[crl-file]]
+=== Crl File
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The path to the CRL file. |  |
+|===
+
+[[Reconnection]]
+=== Reconnection
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> | The reconnection strategy to use. |  |
+|===
+
+[[reconnect]]
+=== Reconnect
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Frequency a| Number | How often in milliseconds to reconnect. | |
+| Count a| Number | How many reconnection attempts to make. | |
+| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+|===
+
+[[reconnect-forever]]
+=== Reconnect Forever
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Frequency a| Number | How often in milliseconds to reconnect. | |
+| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+|===
+
+[[ExpirationPolicy]]
+=== Expiration Policy
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Idle Time a| Number | A scalar time value for the maximum amount of time a dynamic configuration instance should be allowed to be idle before it's considered eligible for expiration |  |
+| Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | A time unit that qualifies the maxIdleTime attribute |  |
+|===
+
+[[AnypointMQMessageAttributes]]
+=== Anypoint MQ Message Attributes
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Destination a| String | The destination the message belongs to. |  | x
+| Headers a| Object | The headers of the message. |  | x
+| Properties a| Object | The properties of the message. |  | x
+| Message Id a| String | The ID of the message. |  | x
+| Delivery Count a| Number | The number of times the message was delivered. |  | x
+| Content Type a| String | The content type of the message body |  |
+| Ack Token a| String | The acknowledgment token of the message, passed to Ack/Nack operations to perform ack/nack manually.
+ This token will be present only when MANUAL ACK mode is used. |  |
+|===
+
+[[circuitBreaker]]
+=== Circuit Breaker
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| On Error Types a| String | The error types whose occurrence during the flow execution will count as a failure in the circuit.
```

Comment:
Error types that equal failure of the circuit when they occur during flow execution. By default, all errors create failure.